apps/studio/src/features/editing-experience/components/PublishButton.tsx

```diff
@@ -50,19 +50,20 @@ const SuspendablePublishButton = ({
     if (coercedSiteId && coercedPageId)
       mutate({ pageId: coercedPageId, siteId: coercedSiteId })
   }
+
+  const isChangesPendingPublish = !!currPage.draftBlobId
```

Comment:
note to self: because we are using this to check, we need to do an implicit publish when page settings are changed (but we won't publish the resource so that's ok)